streamly.cabal

```diff
@@ -615,7 +616,7 @@ library
                      -- streamly-core-data-arrays
                      -- XXX Depends on primitive
                      , Streamly.Internal.Data.Array
-                     , Streamly.Internal.Data.Array.Foreign
+                     , Streamly.Internal.Data.Array.Unboxed
```

Comment:
We have removed the Foreign module completely. Need to only deprecate it, not remove.